content/applications/productivity/whatsapp.rst

```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+- Document
+- Location
+
+Navigate to the :guilabel:`Body` tab to configure the main message of the template.
+
+When all the necessary changes are made to the template, then click on the :guilabel:`Submit for
+approval` button in the upper left hand corner. The status of the template will change to
+:guilabel:`Pending`. The status will remain in :guilabel:`Pending` until a decision has been made by
+Meta. Once the user receives an email confirming that the template has been approved the templates
+will need to be synced from the Odoo database.
+
+See this documentation: :ref:`whatsapp/sync`.
+
+.. tip::
+   Pre-configured demo-data templates can be submitted to Meta for approval. Navigate to
+   :menuselection:`WhatsApp app --> Templates` and select the pre-configured template. Click
+   :guilabel:`Submit for Approval` to start the approval process. An email will be sent to the
+   administrator of the Meta account when the template has been approved.
+
+Using placeholders and variables
+--------------------------------
+
+Dynamic variables reference certain fields within the Odoo database to produce unique data in the
+WhatsApp message when using a template. Dynamic variables are encoded to display fields from within
+the database referencing fields from within a model.
+
+.. example::
+   Many companies like to customize their WhatsApp messages with a personalized piece of customer
+   information to grab attention. This can be accomplished in Odoo by referencing a field within a
+   model by settings a dynamic variable. For example, a customer's name can be referenced in the
+   email from the :guilabel:`Customer` field on the :guilabel:`Sales Order` model.
+
+Dynamic variables can be added in to the :guilabel:`Body` by adding :guilabel:`placeholders` in the
+*text*. To add a placeholder in the *message body* enter the following text `{{1}}`. For the second
+placeholder enter `{{2}}` and  increase incrementally as more placeholders are added to the text.
+
+.. example::
+   Hello {{1}},
+
+   Use the following link to pay *{{2}}{{3}}* to *{{4}}* with reference *{{5}}*.
+   {{6}}
+
+   Don't share screenshots of payment here.
+
+   Thank you
+
+.. seealso::
+   :ref:`whatsapp/templates`
+
+These placeholders must be configured on the :guilabel:`Variables` tab of the template before
+submitting for approval from Meta. To edit the dynamic variables on a template, first change the
+:guilabel:`Type` to :guilabel:`Field of Model`. This allows Odoo to reference a field within a model
+to produce unique data in the message being sent.
+
+Next, go to the :guilabel:`Variables` tab and edit the :guilabel:`Field` field of the dynamic
+variables. The :guilabel:`Applies to` field in the template can be edited prior to ensure the
+correct model and field are referenced.
+
+To search the available fields, type in the front-end name (on user-interface) of the field in the
+search box. This will find a result from all of the available fields for the model
```

Comment:
Again, I don't see where/how to search? I might be missing something but this could be specified/cleared up, because I was lost.

Reply:
Just need to select the box and type in it. A search box appears